pkg/pillar/cmd/zedmanager/handledownloader.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 func AddOrRefcountDownloaderConfig(ctx *zedmanagerContext, safename string,
-	sc types.StorageConfig, ss *types.StorageStatus) {
```

Comment:
Since this function is not modifying the StorageStatus you shouldn't pass a pointer. See earlier discussion on the eve mailing list.

Reply:
sure, will do.